api/utilities.py

```diff
@@ -10,7 +11,9 @@
 from pydantic import TypeAdapter
 
 
-def get_datetime_range(datetime_string: str | None) -> Tuple[Timestamp, Timestamp] | None:
+def get_datetime_range(
+    datetime_string: str | None,
+) -> Tuple[Timestamp, Timestamp] | None:
```

Comment:
Change not needed with 120 line length?

Reply:
Yeah, it's just a local tool getting in the way. Reverted change.